src/Smalot/PdfParser/Document.php

```diff
@@ -198,25 +221,36 @@ public function getFonts()
         return $this->getObjectsByType('Font');
     }
 
+    /**
+     * @return PDFObject
+     */
+    public function getFirstFont()
```

Comment:
What if there are no fonts? Also add return value type.

```suggestion
    public function getFirstFont(): PDFObject
```